src/main/kotlin/lotto/model/NumbersGenerator.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.model
+
+fun interface NumbersGenerator {
+    fun generate(): List<Int>
+}
+
+class RandomNumbersGenerator(private val size: Int = LOTTO_SIZE) : NumbersGenerator {
+    override fun generate(): List<Int> = LOTTO_NUM_RANGE.shuffled().take(size).sorted()
+
```

Comment:
```suggestion
class RandomLotteriesGenerator(val availableCount: Int) : LotteriesGenerator {
    override fun generate(): List<Lotto> = List(availableCount) {
        Lotto(*LOTTO_NUM_RANGE.shuffled().take(LOTTO_SIZE).sorted().toIntArray())
    }
```

가령 그렇게 하면 Controller 의 generateAutoLotteries 메서드의 역할을 매우 간단하게 Generator 로 옮길 수 있습니다.
마찬가지로 ManualLotteriesGenerator 의 구현에 도전해보시기 바랍니다.
ManualLotteriesGenerator 는 RandomLotteriesGenerator 와 전혀 다른 맴버를 가지게 될 수도 있습니다.
그럼에도 view 와는 의존성이 분리되어야 합니다. 어떻게 해야할까요?